CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add additional information from `GCPCluster` to `cluster-values` configmap when running on `gcp`.
 
+### Changed
+
+- The DNS IP needs to come from the cluster `Services` CIDR.
```

Comment:
`kubeadm` uses the 10th IP for the DNS server from the services-cidr (xref: https://github.com/kubernetes/kubernetes/blob/d436f5d0b7eb87f78eb31c12466e2591c24eef59/cmd/kubeadm/app/constants/constants.go#L653-L668)

Reply:
Yeah, [we do the same on this operator](https://github.com/giantswarm/cluster-apps-operator/blob/master/service/controller/key/key.go#L12-L16).